front/app/containers/ProjectsShowPage/shared/header/ProjectInfoSideBar/index.tsx

```diff
@@ -124,300 +126,331 @@ const StyledProjectActionButtons = styled(ProjectActionButtons)`
 
 interface Props {
   projectId: string;
+  hideParticipationNumbers?: boolean;
   className?: string;
 }
 
-const ProjectInfoSideBar = memo<Props>(({ projectId, className }) => {
-  const { data: authUser } = useAuthUser();
-  const { data: project } = useProjectById(projectId, !isAdmin(authUser));
-  const { data: phases } = usePhases(projectId);
-  const { data: events } = useEvents({
-    projectIds: [projectId],
-    sort: '-start_at',
-  });
-
-  const { formatMessage } = useIntl();
-  const [currentPhase, setCurrentPhase] = useState<IPhaseData | undefined>();
-  const [shareModalOpened, setShareModalOpened] = useState(false);
-
-  useEffect(() => {
-    setCurrentPhase(
```

Comment:
@EdwinKato Having a bit of trouble understanding the main changes in this file - is it just the addition of the `hideParticipationNumbers = false }` prop and then its use? :slightly_smiling_face: